README.md

```diff
@@ -72,20 +72,16 @@ See the [toolkit documentation](https://github.com/actions/toolkit/blob/master/R
 
 Actions are run from GitHub repos.  We will create a releases branch and only checkin production modules (core in this case). 
 
-Comment out node_modules in .gitignore and create a releases/v1 branch
-```bash
-# comment out in distribution branches
-# node_modules/
-```
-
+Create a releases/v1 branch:
```

Comment:
We should just link to https://github.com/actions/toolkit/blob/master/docs/action-versioning.md

We stopped doing the `releases/v1` thing because you only need it when master becomes v2. Think of `releases/v1` as a servicing branch... create it when you need it.

And we started setting `npm run build` = `tsc && ncc build` since ncc is fast. Then don't need any explicit verbiage in the readme if the template is setup like that.

Reply:
sounds good. for now i'm going to push this as-is since it will still help people to update it. we can update the template to include ncc by default later

Reply:
I'm going to update this.  @ericsciple is correct that we stopped doing the releases/v1 thing up front.  actions-versioning has the correct guidance.  This doubles your maintenance of master / v1 for no reason (node_modules was the reason before)